lib/internal/Magento/Framework/View/Asset/MergeStrategy/Direct.php

```diff
@@ -30,16 +30,23 @@ class Direct implements \Magento\Framework\View\Asset\MergeStrategyInterface
      */
     private $cssUrlResolver;
 
+    /**
+     * @var \Magento\Framework\Math\Random
+     */
+    protected $mathRandom;
```

Comment:
Hi @Ian410. Thanks for collaboration. According to [Magento Technical Guidelines] we can provide new public/protected properties (https://devdocs.magento.com/guides/v2.3/coding-standards/technical-guidelines.html)

Reply:
Resolved in latest commit.